packages/sdk/src/types/runtime/actor.ts

```diff
@@ -361,7 +320,7 @@ export class Actor implements ActorLike, Patchable<ActorLike> {
 	 * @param colliderType Type of the collider to enable.
 	 * @param isTrigger Whether the collider is a trigger volume or not.
 	 * @param center The center of the collider, or default of the object if none is provided.
-	 * @param size
+	 * @param size The dimensions of the collider.

```

Comment:
>The dimensions of the collider. [](start = 16, length = 31)

Should add in default size here inline with the above comment.

Reply:
Changed the API so the size is not optional.